src/main/java/burp/SettingsTab.form

```diff
@@ -1,146 +1,232 @@
+            <margin top="0" left="0" bottom="0" right="0"/>
+            <constraints>
+              <tabbedpane title="advanced"/>
+            </constraints>
+            <properties>
+              <toolTipText value=""/>
+            </properties>
+            <border type="none"/>
+            <children>
+              <component id="35333" class="javax.swing.JLabel" binding="labelInterceptProxyAddress">
+                <constraints>
+                  <grid row="1" column="0" row-span="1" col-span="1" vsize-policy="0" hsize-policy="0" anchor="8" fill="0" indent="0" use-parent-layout="false"/>
+                </constraints>
+                <properties>
+                  <enabled value="true"/>
+                  <text value="Intercept proxy address:"/>
+                </properties>
+              </component>
+              <component id="1eb46" class="javax.swing.JTextField" binding="textFieldInterceptProxyAddress">
+                <constraints>
+                  <grid row="2" column="0" row-span="1" col-span="1" vsize-policy="0" hsize-policy="6" anchor="8" fill="1" indent="0" use-parent-layout="false"/>
+                </constraints>
+                <properties>
+                  <toolTipText value="Local address the intercept proxy server should listen on. Use it to configure proxy on your client. Requires extension reload."/>
+                </properties>
+              </component>
+              <component id="2d47c" class="javax.swing.JLabel" binding="labelBurpProxyAddress">
+                <constraints>
+                  <grid row="3" column="0" row-span="1" col-span="1" vsize-policy="0" hsize-policy="0" anchor="8" fill="0" indent="0" use-parent-layout="false"/>
+                </constraints>
+                <properties>
+                  <text value="Burp proxy address:"/>
+                </properties>
+              </component>
+              <component id="70622" class="javax.swing.JTextField" binding="textFieldBurpProxyAddress">
+                <constraints>
+                  <grid row="4" column="0" row-span="1" col-span="1" vsize-policy="0" hsize-policy="6" anchor="8" fill="1" indent="0" use-parent-layout="false">
+                    <preferred-size width="150" height="-1"/>
+                  </grid>
+                </constraints>
+                <properties>
+                  <text value=""/>
+                </properties>
+              </component>
+              <component id="4d012" class="javax.swing.JRadioButton" binding="radioButtonUseInterceptedFingerprint">
+                <constraints>
+                  <grid row="0" column="0" row-span="1" col-span="1" vsize-policy="0" hsize-policy="3" anchor="8" fill="0" indent="0" use-parent-layout="false"/>
+                </constraints>
+                <properties>
+                  <text value="Use intercepted tls fingerprint"/>
+                </properties>
+              </component>
```

Comment:
Is this radio button uncheckable (to turn the intercept proxy feature off)? If not, can you make this a checkbox instead?

Reply:
No, you can turn it on and off.

Reply:
fixed

Reply:
Don't forget to update your screenshots too if you haven't done so already!